generators/server/templates/src/test/java/package/config/CassandraTestContainerExtension.java.ejs

```diff
@@ -0,0 +1,132 @@
+package <%= packageName %>.config;
+
+import static java.nio.file.Files.newDirectoryStream;
+import static java.nio.file.Paths.get;
+import static java.util.Spliterator.SORTED;
+import static java.util.Spliterators.spliteratorUnknownSize;
+import static java.util.stream.StreamSupport.stream;
+
+import com.datastax.driver.core.Cluster;
+import com.datastax.driver.core.Session;
+import com.datastax.oss.driver.api.core.CqlSession;
+import java.io.IOException;
+import java.net.InetSocketAddress;
+import java.net.URISyntaxException;
+import java.net.URL;
+import java.nio.file.Path;
+import java.time.Duration;
+import java.time.temporal.ChronoUnit;
+import java.util.Iterator;
+import java.util.Spliterator;
+
+import com.datastax.oss.driver.api.core.CqlSessionBuilder;
+import com.datastax.oss.driver.api.core.config.DefaultDriverOption;
+import com.datastax.oss.driver.api.core.config.DriverConfigLoader;
+import org.cassandraunit.CQLDataLoader;
+import org.cassandraunit.dataset.cql.ClassPathCQLDataSet;
+import org.junit.jupiter.api.extension.AfterAllCallback;
+import org.junit.jupiter.api.extension.BeforeAllCallback;
+import org.junit.jupiter.api.extension.ExtensionContext;
+import org.springframework.context.ApplicationContext;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.testcontainers.containers.CassandraContainer;
+import org.testcontainers.containers.output.Slf4jLogConsumer;
+
+/**
+ * Base class for starting/stopping Cassandra during tests.
+ */
+public class CassandraTestContainerExtension implements BeforeAllCallback, AfterAllCallback {
```

Comment:
Conditional on testcontainers profile?

Reply:
I would prefer to drop the testcontainers profile and make it standard.
-> If the user chooses something that can be used with testcontainer (everything but h2) == testcontainer
-> Else depends on the maven profile (dev or prod)

WDYT?

Reply:
And still, Cassandra testcontainer is a must have for testing stacks on cassandra